case_studies/sdss_galaxies/Galaxy Flow.ipynb

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
Do you think these two notebooks are useful to keep in the repo? Just from skimming them it seems that these notebooks demonstrate that the VAE and the flow galaxy samples look reasonable right?

Reply:
No, I don't want to clutter the repo with these - I must have committed them by accident. I went ahead and removed them